src/Draw.cpp

```diff
@@ -37,36 +37,47 @@ typedef unsigned char guint8;
 whichPart_t CDraw::m_displayHand;
 int CDraw::m_forceCompileRedraw;
 
+FTFont * setupFont() {
+    static FTFont * cacheFont = NULL;
```

Comment:
As mentioned in the other comment, other places `delete` this object expecting to take over ownership. However, only one of the possible callers can actually have ownership. Maybe the most elegant solution is:

```suggestion
    static FTFont cacheFont = [] {
        // compute params for FTBitmapFont constructor
        …
        // possibly do an early-exit
        …

        return FTBitmapFont(…);
    };
```

Then you can just `return &cacheFont` and make sure all places using `setupFont()` do *not* take ownership.

Reply:
Thanks Martchus.  Using your suggestion of adding a debug build profile and adding an additional flag for debug, I was able to debug the code.  Good catch on that cached font.  It sure can cause issue.  I just removed the static flag for now.  I don't know how much overhead it may have, so at least with limited time, I am not going to play with it, and leave it for someone or you to take care of it.  Various issue like multi-threading, etc., I just don't want to touch beyond what I originally intended to do.  Thanks again for all the great feedback. I think it made a big difference.

Reply:
Guys before you get too excited about this. I think that themes are the way to go on this. So I would be very grateful if you all choose a nice set of colours that works well for you and a nice theme name and have a go adding the code to add themes. If we get a nice set of themes most users won't need to edit the colours themselves. To add a new theme with custom colours I favour defining themes in either a separate text config file or putting them in the standard pb config file. One of my aims of PB is to keep the UI really clean and simple so that even young children can use the program. Some young children (and even some adults) might not even know or understand what RGB is. The difficulty allowing unskilled users to control the colours is what happens if a user accidentally sets the background colour the same colour as the stave lines?

I am away at the moment and very busy so sorry I haven't been able to review anything yet. But I think we need themes implemented before we add any UI to add custom colours which I am undecided which is best custom colour UI or via a text file.